db/schema.rb

```diff
@@ -525,6 +525,12 @@
     t.datetime "updated_at",                null: false
   end
 
+  create_table "featured_policies", force: :cascade do |t|
+    t.string  "policy_content_id", limit: 255
+    t.integer "ordering",          limit: 4
```

Comment:
Something else just occurred to me; we wound up in a bunch of trouble with attachment orderings a while back which was exacerbated by allowing the ordering field to be `null`.  Since then we've had constraints on those fields to [require an ordering](https://github.com/alphagov/whitehall/blob/master/db/schema.rb#L66).